data.txt

```diff
@@ -118,8 +118,8 @@
   last_updated: 2016-12-18
 [reddit]
   company: Reddit
-  num_female_eng: 2
-  num_eng: 14
+  num_female_eng: 3
+  num_eng: 66
```

Comment:
On further thought -- LinkedIn isn't a very reliable source of data. Do you have a better source or confirmation otherwise?